src/sentry/utils/performance_issues/performance_problem.py

```diff
@@ -15,6 +16,12 @@ class PerformanceProblem:
     cause_span_ids: Optional[Sequence[str]]
     # The actual bad spans
     offender_span_ids: Sequence[str]
+    # Evidence to be used for the group
+    # TODO make it required once all detectors have been migrated to platform
```

Comment:
Just to add to the TODO: We can't make it required until we stop loading these from nodestore via `EventPerformanceProblem`, since there's legacy data in there that won't have these fields. So until we disable transaction based perf issues we'll need to keep this optional.

Reply:
added!